src/telliot_feeds/feeds/__init__.py

```diff
@@ -53,13 +58,14 @@
     "eur-usd-spot": eur_usd_median_feed,
 }
 
-DATAFEED_BUILDER_MAPPING = {
+DATAFEED_BUILDER_MAPPING: Dict[str, DataFeed[Any]] = {
     # "spotprice",
     # "apiquery",
     # "divaprotocolpolygon",
     "GasPriceOracle": gas_price_oracle_feed,
+    "NumericApiResponse": numeric_api_response_feed,
```

Comment:
Move all feeds you've made with parameters set as `None` to a folder/module within `telliot_feeds.feeds` and call it unset_params or no_params or something.

I feel like there's a way to dynamically initialize any of these with unset params instead of making a file for each, but for now let's just make them and organize them in their own folder..

Reply:
maybe for a separate issue?